x-pack/plugins/triggers_actions_ui/public/application/components/prompts/empty_connectors_prompt.tsx

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { FormattedMessage } from '@kbn/i18n/react';
+import React, { Fragment } from 'react';
+import { EuiButton, EuiEmptyPrompt, EuiIcon, EuiSpacer, EuiTitle } from '@elastic/eui';
+
+export const EmptyConnectorsPrompt = ({ onCTAClicked }: { onCTAClicked: () => void }) => (
+  <EuiEmptyPrompt
+    data-test-subj="createFirstConnectorEmptyPrompt"
+    title={
+      <Fragment>
+        <EuiIcon type="logoSlack" size="xl" className="actConnectorsList__logo" />
+        <EuiIcon type="logoGmail" size="xl" className="actConnectorsList__logo" />
+        <EuiIcon type="logoWebhook" size="xl" className="actConnectorsList__logo" />
```

Comment:
It looks like we should extract some SCSS from `x-pack/plugins/triggers_actions_ui/public/application/sections/actions_connectors_list/components/actions_connectors_list.scss` and move it within a sibling file here.

Maybe rename the class to `actEmptyConnectorsPrompt__logo`